src/DependencyInjection/Configuration.php

```diff
@@ -181,6 +181,12 @@ public function getConfigTreeBuilder()
             ->end()
 
             ->arrayNode('assets')
+                ->setDeprecated(
+                    ...$this->getDeprecationMessage(
+                        'The "assets" option is deprecated since sonata-project/page-bundle 3.28.0 and will be removed in 4.0',
```

Comment:
```suggestion
                        'The "assets" option is deprecated since sonata-project/page-bundle 3.29.0 and will be removed in 4.0',
```

Reply:
Shouldn't this be mentioned in the upgrade-3.x guide?

Reply:
I can add a note for this :)